packages/docusaurus-theme-classic/src/types.d.ts

```diff
@@ -396,6 +396,18 @@ declare module '@theme/Tabs' {
   export default Tabs;
 }
 
+declare module '@theme/ThemedImage' {
+  import type {ComponentProps} from 'react';
+
+  export type Props = {
+    readonly lightSrc: string;
+    readonly darkSrc?: string;
+  } & ComponentProps<'img'>;
+
+  const ThemedImage: () => JSX.Element;
```

Comment:
shouldn't you use the props here?

Reply:
?

Reply:
const ThemedImage: (props: Props) => JSX.Element;

Reply:
This has been taken from the `Tabs` component types already in that file (https://github.com/facebook/docusaurus/blob/master/packages/docusaurus-theme-classic/src/types.d.ts#L395):
```
  const Tabs: () => JSX.Element;
  export default Tabs;
```

TS should be happy with that construction because this method only serves as the definition for the returned value.